src/java_bytecode/CMakeLists.txt

```diff
@@ -1,19 +1,20 @@
-get_filename_component(JAVA_CORE_LIBRARY_PATH "library" REALPATH BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+find_package(Java REQUIRED)
+include(UseJava)
```

Comment:
Commit can be squashed into https://github.com/cesaro/cbmc/pull/5/commits/068c730d5164434a1b3c5894d57b04e361cef962